settings.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
we're gonna store important settings in that file.
whenever we'd need to use these variables we're just importing the module ```import settings```